packages/lexical-editor/src/nodes/webinyNodes.ts

```diff
@@ -11,6 +11,8 @@ import { TypographyElementNode } from "~/nodes/TypographyElementNode";
 import { WebinyListNode } from "~/nodes/list-node/WebinyListNode";
 import { WebinyListItemNode } from "~/nodes/list-node/WebinyListItemNode";
 import { WebinyQuoteNode } from "~/nodes/WebinyQuoteNode";
+import { BaseParagraphNode } from "~/nodes/BaseParagraphNode";
+import { ParagraphNode } from "lexical";
 
 export const WebinyNodes: ReadonlyArray<
```

Comment:
Let's also add a comment here, saying this is a list of all of the nodes that Webiny's Lexical implementation supports OOTB.